### ISSUE TITLE
Users control on team size in task-DAG

### ISSUE THREAD

**Issue by kyungjoo-kim** [kokkos/kokkos#2685]
On Cuda, Kokkos tasking maps tasks into a single warp (32 threads). It would be nice if users can control the team size for tasking. With discussion with @dhollman and @crtrott, it is possible to adjust the team size within a single SM. The feature I would like to have to control both team and vector size.

**Comment by kyungjoo-kim** [comment]
@dhollman Can I ping about this ?

**Comment by dhollman** [comment]
Yeah, I'm sorry, I haven't gotten around to this. I haven't had time to do anything in Kokkos in the month of January, though...

**Comment by cwpearson** [comment]
Tasking removed in #8390
